startleft/startleft/cli/cli.py

```diff
@@ -141,45 +145,38 @@ def parse_etm(etm_type, default_mapping_file, custom_mapping_file, output_file,
               help=IAC_TYPE_DESC,
               cls=Exclusion,
               mandatory=True,
-              mutually_exclusion=['diagram_type', 'etm_type', 'default_mapping_file', 'custom_mapping_file'])
+              mutually_exclusion=['diagram_type', 'etm_type'])
 @click.option(DIAGRAM_TYPE_NAME, DIAGRAM_TYPE_SHORTNAME,
               type=click.Choice(DIAGRAM_TYPE_SUPPORTED, case_sensitive=False),
               help=DIAGRAM_TYPE_DESC,
               cls=Exclusion,
               mandatory=True,
-              mutually_exclusion=['iac_type', 'etm_type', 'mapping_file'])
+              mutually_exclusion=['iac_type', 'etm_type'])
 @click.option(ETM_TYPE_NAME, ETM_TYPE_SHORTNAME,
               type=click.Choice(ETM_TYPE_SUPPORTED, case_sensitive=False),
               help=ETM_TYPE_DESC,
               cls=Exclusion,
               mandatory=True,
-              mutually_exclusion=['diagram_type', 'iac_type', 'mapping_file'])
-@click.option(MAPPING_FILE_NAME, MAPPING_FILE_SHORTNAME,
-              help=MAPPING_FILE_DESC,
-              cls=Exclusion,
-              mandatory=True,
-              mutually_exclusion=['default_mapping_file', 'custom_mapping_file', 'diagram_type', 'etm_type'])
+              mutually_exclusion=['diagram_type', 'iac_type'])
 @click.option(DEFAULT_MAPPING_FILE_NAME, DEFAULT_MAPPING_FILE_SHORTNAME,
               help=DEFAULT_MAPPING_FILE_DESC,
               cls=Exclusion,
-              mandatory=True,
-              mutually_exclusion=['mapping_file', 'iac_type'])
+              mandatory=True)
```

Comment:
Since the default_mapping has not mutually_exclusion anymore, the mandatory field is not needed, however the default_mapping  is a required field, so it should has the required field a True.
By this way the --help option will show the field as [required]
I would change here the mandatory=True by required=True

Reply:
Yep, I missed it. Thank you! Already fixed.